--- php
+++ phl
@@ -1,5 +1,3 @@
-Xdebug: [Config] The setting 'xdebug.overload_var_dump' has been removed, see the upgrading guide at https://xdebug.org/docs/upgrade_guide#changed-xdebug.overload_var_dump (See: https://xdebug.org/docs/errors#CFG-C-REMOVED)
-Xdebug: [Config] The setting 'xdebug.overload_var_dump' has been removed, see the upgrading guide at https://xdebug.org/docs/upgrade_guide#changed-xdebug.overload_var_dump (See: https://xdebug.org/docs/errors#CFG-C-REMOVED)
 PHPUnit 9.6.36 by Sebastian Bergmann and contributors.
 
 Testing
@@ -8,7 +6,7 @@
 .............................................................  183 / 1681 ( 10%)
 .............................................................  244 / 1681 ( 14%)
 ...............................................S.............  305 / 1681 ( 18%)
-.....................S.S.....................................  366 / 1681 ( 21%)
+.............................................................  366 / 1681 ( 21%)
 ............................S................................  427 / 1681 ( 25%)
 .............................................................  488 / 1681 ( 29%)
 .............................................................  549 / 1681 ( 32%)
@@ -35,12 +33,12 @@
 Time: @TIME@
 
 OK, but incomplete, skipped, or risky tests!
-Tests: 1681, Assertions: 4401, Skipped: 7.
+Tests: 1681, Assertions: 4404, Skipped: 5.
 
-Remaining self deprecation notices (463)
+Remaining self deprecation notices (465)
 
-  442x: Method ReflectionProperty::setAccessible() is deprecated since 8.5, as it has no effect since PHP 8.1
-    367x in IntegrationTest::testIntegration from Twig\Tests
+  444x: Method ReflectionProperty::setAccessible() is deprecated since 8.5, as it has no effect since PHP 8.1
+    369x in IntegrationTest::testIntegration from Twig\Tests
     22x in BlackfireTest::testDump from Twig\Tests\Profiler\Dumper
     22x in HtmlTest::testDump from Twig\Tests\Profiler\Dumper
     22x in TextTest::testDump from Twig\Tests\Profiler\Dumper
@@ -59,4 +57,4 @@
     3x in TemplateTest::testGetAttributeOnArrayWithConfusableKey from Twig\Tests
 
 Legacy deprecation notices (8)
-@exit@ 1
+@exit@ 0
