--- php
+++ phl
@@ -1,5 +1,3 @@
-Xdebug: [Config] The setting 'xdebug.overload_var_dump' has been removed, see the upgrading guide at https://xdebug.org/docs/upgrade_guide#changed-xdebug.overload_var_dump (See: https://xdebug.org/docs/errors#CFG-C-REMOVED)
-Xdebug: [Config] The setting 'xdebug.overload_var_dump' has been removed, see the upgrading guide at https://xdebug.org/docs/upgrade_guide#changed-xdebug.overload_var_dump (See: https://xdebug.org/docs/errors#CFG-C-REMOVED)
 PHPUnit 9.6.36 by Sebastian Bergmann and contributors.
 
 Testing
@@ -7,9 +5,838 @@
 .............................................................  122 / 1681 (  7%)
 .............................................................  183 / 1681 ( 10%)
 .............................................................  244 / 1681 ( 14%)
-...............................................S.............  305 / 1681 ( 18%)
-.....................S.S.....................................  366 / 1681 ( 21%)
-............................S................................  427 / 1681 ( 25%)
+...............................................S.FCompiled templates that failed on case 1:
+Template: index.twig
+<?php
+
+use Twig\Environment;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
+use Twig\Extension\CoreExtension;
+use Twig\Extension\SandboxExtension;
+use Twig\Markup;
+use Twig\Sandbox\SecurityError;
+use Twig\Sandbox\SecurityNotAllowedTagError;
+use Twig\Sandbox\SecurityNotAllowedFilterError;
+use Twig\Sandbox\SecurityNotAllowedFunctionError;
+use Twig\Source;
+use Twig\Template;
+
+/* index.twig */
+class __TwigTemplate_@HASH@_024d90e049fc69c87dcff025580ad1c4 extends Template
+{
+    private $source;
+    private $macros = [];
+
+    public function __construct(Environment $env)
+    {
+        parent::__construct($env);
+
+        $this->source = $this->getSourceContext();
+
+        $this->parent = false;
+
+        $this->blocks = [
+        ];
+        $this->sandbox = $this->env->getExtension('\Twig\Extension\SandboxExtension');
+        $this->checkSecurity();
+    }
+
+    protected function doDisplay(array $context, array $blocks = [])
+    {
+        $macros = $this->macros;
+        // line 1
+        yield "
+";
+        // line 2
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, Twig\Extension\CoreExtension::dateConverter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date"]) || array_key_exists("date", $context) ? $context["date"] : (function () { throw new RuntimeError('Variable "date" does not exist.', 2, $this->source); })()), 2, $this->source), "America/New_York"), "d/m/Y H:i:s P", false), "html", null, true);
+        yield "
+";
+        // line 3
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, Twig\Extension\CoreExtension::dateConverter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date"]) || array_key_exists("date", $context) ? $context["date"] : (function () { throw new RuntimeError('Variable "date" does not exist.', 3, $this->source); })()), 3, $this->source), "America/New_York"), "d/m/Y H:i:s P", false), "html", null, true);
+        return; yield '';
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function getTemplateName()
+    {
+        return "index.twig";
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function isTraitable()
+    {
+        return false;
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function getDebugInfo()
+    {
+        return array (  47 => 3,  43 => 2,  40 => 1,);
+    }
+
+    public function getSourceContext()
+    {
+        return new Source("", "index.twig", "");
+    }
+
+    public function checkSecurity()
+    {
+        static $tags = array();
+        static $filters = array("escape" => 2, "date" => 2);
+        static $functions = array("date" => 2);
+
+        try {
+            $this->sandbox->checkSecurity(
+                [],
+                ['escape', 'date'],
+                ['date'],
+                $this->source
+            );
+        } catch (SecurityError $e) {
+            $e->setSourceContext($this->source);
+
+            if ($e instanceof SecurityNotAllowedTagError && isset($tags[$e->getTagName()])) {
+                $e->setTemplateLine($tags[$e->getTagName()]);
+            } elseif ($e instanceof SecurityNotAllowedFilterError && isset($filters[$e->getFilterName()])) {
+                $e->setTemplateLine($filters[$e->getFilterName()]);
+            } elseif ($e instanceof SecurityNotAllowedFunctionError && isset($functions[$e->getFunctionName()])) {
+                $e->setTemplateLine($functions[$e->getFunctionName()]);
+            }
+
+            throw $e;
+        }
+
+    }
+}
+...........  305 / 1681 ( 18%)
+.......................................FCompiled templates that failed on case 1:
+Template: index.twig
+<?php
+
+use Twig\Environment;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
+use Twig\Extension\CoreExtension;
+use Twig\Extension\SandboxExtension;
+use Twig\Markup;
+use Twig\Sandbox\SecurityError;
+use Twig\Sandbox\SecurityNotAllowedTagError;
+use Twig\Sandbox\SecurityNotAllowedFilterError;
+use Twig\Sandbox\SecurityNotAllowedFunctionError;
+use Twig\Source;
+use Twig\Template;
+
+/* index.twig */
+class __TwigTemplate_@HASH@_4360e0675c5c37e62e3c6c559cba6010 extends Template
+{
+    private $source;
+    private $macros = [];
+
+    public function __construct(Environment $env)
+    {
+        parent::__construct($env);
+
+        $this->source = $this->getSourceContext();
+
+        $this->parent = false;
+
+        $this->blocks = [
+        ];
+        $this->sandbox = $this->env->getExtension('\Twig\Extension\SandboxExtension');
+        $this->checkSecurity();
+    }
+
+    protected function doDisplay(array $context, array $blocks = [])
+    {
+        $macros = $this->macros;
+        // line 1
+        yield "
+";
+        // line 2
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date"]) || array_key_exists("date", $context) ? $context["date"] : (function () { throw new RuntimeError('Variable "date" does not exist.', 2, $this->source); })()), 2, $this->source), "d/m/Y H:i:s P", "America/Chicago"), "html", null, true);
+        yield "
+";
+        // line 3
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date"]) || array_key_exists("date", $context) ? $context["date"] : (function () { throw new RuntimeError('Variable "date" does not exist.', 3, $this->source); })()), 3, $this->source), "d/m/Y H:i:s P", "America/Chicago"), "html", null, true);
+        yield "
+";
+        // line 4
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date"]) || array_key_exists("date", $context) ? $context["date"] : (function () { throw new RuntimeError('Variable "date" does not exist.', 4, $this->source); })()), 4, $this->source), "d/m/Y H:i:s P", "America/Chicago"), "html", null, true);
+        return; yield '';
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function getTemplateName()
+    {
+        return "index.twig";
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function isTraitable()
+    {
+        return false;
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function getDebugInfo()
+    {
+        return array (  51 => 4,  47 => 3,  43 => 2,  40 => 1,);
+    }
+
+    public function getSourceContext()
+    {
+        return new Source("", "index.twig", "");
+    }
+
+    public function checkSecurity()
+    {
+        static $tags = array();
+        static $filters = array("escape" => 2, "date" => 2);
+        static $functions = array();
+
+        try {
+            $this->sandbox->checkSecurity(
+                [],
+                ['escape', 'date'],
+                [],
+                $this->source
+            );
+        } catch (SecurityError $e) {
+            $e->setSourceContext($this->source);
+
+            if ($e instanceof SecurityNotAllowedTagError && isset($tags[$e->getTagName()])) {
+                $e->setTemplateLine($tags[$e->getTagName()]);
+            } elseif ($e instanceof SecurityNotAllowedFilterError && isset($filters[$e->getFilterName()])) {
+                $e->setTemplateLine($filters[$e->getFilterName()]);
+            } elseif ($e instanceof SecurityNotAllowedFunctionError && isset($functions[$e->getFunctionName()])) {
+                $e->setTemplateLine($functions[$e->getFunctionName()]);
+            }
+
+            throw $e;
+        }
+
+    }
+}
+............FCompiled templates that failed on case 1:
+Template: index.twig
+<?php
+
+use Twig\Environment;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
+use Twig\Extension\CoreExtension;
+use Twig\Extension\SandboxExtension;
+use Twig\Markup;
+use Twig\Sandbox\SecurityError;
+use Twig\Sandbox\SecurityNotAllowedTagError;
+use Twig\Sandbox\SecurityNotAllowedFilterError;
+use Twig\Sandbox\SecurityNotAllowedFunctionError;
+use Twig\Source;
+use Twig\Template;
+
+/* index.twig */
+class __TwigTemplate_@HASH@_ca5cc2f65ec1c1df679a76d920956abb extends Template
+{
+    private $source;
+    private $macros = [];
+
+    public function __construct(Environment $env)
+    {
+        parent::__construct($env);
+
+        $this->source = $this->getSourceContext();
+
+        $this->parent = false;
+
+        $this->blocks = [
+        ];
+        $this->sandbox = $this->env->getExtension('\Twig\Extension\SandboxExtension');
+        $this->checkSecurity();
+    }
+
+    protected function doDisplay(array $context, array $blocks = [])
+    {
+        $macros = $this->macros;
+        // line 1
+        yield "
+";
+        // line 2
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 2, $this->source); })()), 2, $this->source)), "html", null, true);
+        yield "
+";
+        // line 3
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 3, $this->source); })()), 3, $this->source), "%d days %h hours"), "html", null, true);
+        yield "
+";
+        // line 4
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 4, $this->source); })()), 4, $this->source), "%d days %h hours", $this->sandbox->ensureToStringAllowed((isset($context["timezone1"]) || array_key_exists("timezone1", $context) ? $context["timezone1"] : (function () { throw new RuntimeError('Variable "timezone1" does not exist.', 4, $this->source); })()), 4, $this->source)), "html", null, true);
+        return; yield '';
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function getTemplateName()
+    {
+        return "index.twig";
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function isTraitable()
+    {
+        return false;
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function getDebugInfo()
+    {
+        return array (  51 => 4,  47 => 3,  43 => 2,  40 => 1,);
+    }
+
+    public function getSourceContext()
+    {
+        return new Source("", "index.twig", "");
+    }
+
+    public function checkSecurity()
+    {
+        static $tags = array();
+        static $filters = array("escape" => 2, "date" => 2);
+        static $functions = array();
+
+        try {
+            $this->sandbox->checkSecurity(
+                [],
+                ['escape', 'date'],
+                [],
+                $this->source
+            );
+        } catch (SecurityError $e) {
+            $e->setSourceContext($this->source);
+
+            if ($e instanceof SecurityNotAllowedTagError && isset($tags[$e->getTagName()])) {
+                $e->setTemplateLine($tags[$e->getTagName()]);
+            } elseif ($e instanceof SecurityNotAllowedFilterError && isset($filters[$e->getFilterName()])) {
+                $e->setTemplateLine($filters[$e->getFilterName()]);
+            } elseif ($e instanceof SecurityNotAllowedFunctionError && isset($functions[$e->getFunctionName()])) {
+                $e->setTemplateLine($functions[$e->getFunctionName()]);
+            }
+
+            throw $e;
+        }
+
+    }
+}
+FCompiled templates that failed on case 1:
+Template: index.twig
+<?php
+
+use Twig\Environment;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
+use Twig\Extension\CoreExtension;
+use Twig\Extension\SandboxExtension;
+use Twig\Markup;
+use Twig\Sandbox\SecurityError;
+use Twig\Sandbox\SecurityNotAllowedTagError;
+use Twig\Sandbox\SecurityNotAllowedFilterError;
+use Twig\Sandbox\SecurityNotAllowedFunctionError;
+use Twig\Source;
+use Twig\Template;
+
+/* index.twig */
+class __TwigTemplate_@HASH@_fbfede61c61f24be785ef86574ba7256 extends Template
+{
+    private $source;
+    private $macros = [];
+
+    public function __construct(Environment $env)
+    {
+        parent::__construct($env);
+
+        $this->source = $this->getSourceContext();
+
+        $this->parent = false;
+
+        $this->blocks = [
+        ];
+        $this->sandbox = $this->env->getExtension('\Twig\Extension\SandboxExtension');
+        $this->checkSecurity();
+    }
+
+    protected function doDisplay(array $context, array $blocks = [])
+    {
+        $macros = $this->macros;
+        // line 1
+        yield "
+";
+        // line 2
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 2, $this->source); })()), 2, $this->source)), "html", null, true);
+        yield "
+";
+        // line 3
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 3, $this->source); })()), 3, $this->source), "d/m/Y"), "html", null, true);
+        yield "
+";
+        // line 4
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 4, $this->source); })()), 4, $this->source), "d/m/Y H:i:s", "Asia/Hong_Kong"), "html", null, true);
+        yield "
+";
+        // line 5
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 5, $this->source); })()), 5, $this->source), "d/m/Y H:i:s", $this->sandbox->ensureToStringAllowed((isset($context["timezone1"]) || array_key_exists("timezone1", $context) ? $context["timezone1"] : (function () { throw new RuntimeError('Variable "timezone1" does not exist.', 5, $this->source); })()), 5, $this->source)), "html", null, true);
+        yield "
+";
+        // line 6
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 6, $this->source); })()), 6, $this->source), "d/m/Y H:i:s"), "html", null, true);
+        yield "
+";
+        // line 7
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, Twig\Extension\CoreExtension::dateModifyFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 7, $this->source); })()), 7, $this->source), "+1 hour"), "d/m/Y H:i:s"), "html", null, true);
+        yield "
+
+";
+        // line 9
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date2"]) || array_key_exists("date2", $context) ? $context["date2"] : (function () { throw new RuntimeError('Variable "date2" does not exist.', 9, $this->source); })()), 9, $this->source), "d/m/Y H:i:s P", "Europe/Paris"), "html", null, true);
+        yield "
+";
+        // line 10
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date2"]) || array_key_exists("date2", $context) ? $context["date2"] : (function () { throw new RuntimeError('Variable "date2" does not exist.', 10, $this->source); })()), 10, $this->source), "d/m/Y H:i:s P", "Asia/Hong_Kong"), "html", null, true);
+        yield "
+";
+        // line 11
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date2"]) || array_key_exists("date2", $context) ? $context["date2"] : (function () { throw new RuntimeError('Variable "date2" does not exist.', 11, $this->source); })()), 11, $this->source), "d/m/Y H:i:s P", false), "html", null, true);
+        yield "
+";
+        // line 12
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date2"]) || array_key_exists("date2", $context) ? $context["date2"] : (function () { throw new RuntimeError('Variable "date2" does not exist.', 12, $this->source); })()), 12, $this->source), "e", "Europe/Paris"), "html", null, true);
+        yield "
+";
+        // line 13
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date2"]) || array_key_exists("date2", $context) ? $context["date2"] : (function () { throw new RuntimeError('Variable "date2" does not exist.', 13, $this->source); })()), 13, $this->source), "e", false), "html", null, true);
+        return; yield '';
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function getTemplateName()
+    {
+        return "index.twig";
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function isTraitable()
+    {
+        return false;
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function getDebugInfo()
+    {
+        return array (  84 => 13,  80 => 12,  76 => 11,  72 => 10,  68 => 9,  63 => 7,  59 => 6,  55 => 5,  51 => 4,  47 => 3,  43 => 2,  40 => 1,);
+    }
+
+    public function getSourceContext()
+    {
+        return new Source("", "index.twig", "");
+    }
+
+    public function checkSecurity()
+    {
+        static $tags = array();
+        static $filters = array("escape" => 2, "date" => 2, "date_modify" => 7);
+        static $functions = array();
+
+        try {
+            $this->sandbox->checkSecurity(
+                [],
+                ['escape', 'date', 'date_modify'],
+                [],
+                $this->source
+            );
+        } catch (SecurityError $e) {
+            $e->setSourceContext($this->source);
+
+            if ($e instanceof SecurityNotAllowedTagError && isset($tags[$e->getTagName()])) {
+                $e->setTemplateLine($tags[$e->getTagName()]);
+            } elseif ($e instanceof SecurityNotAllowedFilterError && isset($filters[$e->getFilterName()])) {
+                $e->setTemplateLine($filters[$e->getFilterName()]);
+            } elseif ($e instanceof SecurityNotAllowedFunctionError && isset($functions[$e->getFunctionName()])) {
+                $e->setTemplateLine($functions[$e->getFunctionName()]);
+            }
+
+            throw $e;
+        }
+
+    }
+}
+...FCompiled templates that failed on case 1:
+Template: index.twig
+<?php
+
+use Twig\Environment;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
+use Twig\Extension\CoreExtension;
+use Twig\Extension\SandboxExtension;
+use Twig\Markup;
+use Twig\Sandbox\SecurityError;
+use Twig\Sandbox\SecurityNotAllowedTagError;
+use Twig\Sandbox\SecurityNotAllowedFilterError;
+use Twig\Sandbox\SecurityNotAllowedFunctionError;
+use Twig\Source;
+use Twig\Template;
+
+/* index.twig */
+class __TwigTemplate_@HASH@_d54f94b53ebf6daffaaf4e5c2875e130 extends Template
+{
+    private $source;
+    private $macros = [];
+
+    public function __construct(Environment $env)
+    {
+        parent::__construct($env);
+
+        $this->source = $this->getSourceContext();
+
+        $this->parent = false;
+
+        $this->blocks = [
+        ];
+        $this->sandbox = $this->env->getExtension('\Twig\Extension\SandboxExtension');
+        $this->checkSecurity();
+    }
+
+    protected function doDisplay(array $context, array $blocks = [])
+    {
+        $macros = $this->macros;
+        // line 1
+        yield "
+";
+        // line 2
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 2, $this->source); })()), 2, $this->source)), "html", null, true);
+        yield "
+";
+        // line 3
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 3, $this->source); })()), 3, $this->source), "d/m/Y"), "html", null, true);
+        yield "
+";
+        // line 4
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 4, $this->source); })()), 4, $this->source), "d/m/Y H:i:s", "Asia/Hong_Kong"), "html", null, true);
+        yield "
+";
+        // line 5
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 5, $this->source); })()), 5, $this->source), "d/m/Y H:i:s P", "Asia/Hong_Kong"), "html", null, true);
+        yield "
+";
+        // line 6
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 6, $this->source); })()), 6, $this->source), "d/m/Y H:i:s P", "America/Chicago"), "html", null, true);
+        yield "
+";
+        // line 7
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 7, $this->source); })()), 7, $this->source), "e"), "html", null, true);
+        yield "
+";
+        // line 8
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date1"]) || array_key_exists("date1", $context) ? $context["date1"] : (function () { throw new RuntimeError('Variable "date1" does not exist.', 8, $this->source); })()), 8, $this->source), "d/m/Y H:i:s"), "html", null, true);
+        yield "
+
+";
+        // line 10
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date2"]) || array_key_exists("date2", $context) ? $context["date2"] : (function () { throw new RuntimeError('Variable "date2" does not exist.', 10, $this->source); })()), 10, $this->source)), "html", null, true);
+        yield "
+";
+        // line 11
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date2"]) || array_key_exists("date2", $context) ? $context["date2"] : (function () { throw new RuntimeError('Variable "date2" does not exist.', 11, $this->source); })()), 11, $this->source), "d/m/Y"), "html", null, true);
+        yield "
+";
+        // line 12
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date2"]) || array_key_exists("date2", $context) ? $context["date2"] : (function () { throw new RuntimeError('Variable "date2" does not exist.', 12, $this->source); })()), 12, $this->source), "d/m/Y H:i:s", "Asia/Hong_Kong"), "html", null, true);
+        yield "
+";
+        // line 13
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date2"]) || array_key_exists("date2", $context) ? $context["date2"] : (function () { throw new RuntimeError('Variable "date2" does not exist.', 13, $this->source); })()), 13, $this->source), "d/m/Y H:i:s", $this->sandbox->ensureToStringAllowed((isset($context["timezone1"]) || array_key_exists("timezone1", $context) ? $context["timezone1"] : (function () { throw new RuntimeError('Variable "timezone1" does not exist.', 13, $this->source); })()), 13, $this->source)), "html", null, true);
+        yield "
+";
+        // line 14
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date2"]) || array_key_exists("date2", $context) ? $context["date2"] : (function () { throw new RuntimeError('Variable "date2" does not exist.', 14, $this->source); })()), 14, $this->source), "d/m/Y H:i:s"), "html", null, true);
+        yield "
+
+";
+        // line 16
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date3"]) || array_key_exists("date3", $context) ? $context["date3"] : (function () { throw new RuntimeError('Variable "date3" does not exist.', 16, $this->source); })()), 16, $this->source)), "html", null, true);
+        yield "
+";
+        // line 17
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date3"]) || array_key_exists("date3", $context) ? $context["date3"] : (function () { throw new RuntimeError('Variable "date3" does not exist.', 17, $this->source); })()), 17, $this->source), "d/m/Y"), "html", null, true);
+        yield "
+
+";
+        // line 19
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date4"]) || array_key_exists("date4", $context) ? $context["date4"] : (function () { throw new RuntimeError('Variable "date4" does not exist.', 19, $this->source); })()), 19, $this->source)), "html", null, true);
+        yield "
+";
+        // line 20
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date4"]) || array_key_exists("date4", $context) ? $context["date4"] : (function () { throw new RuntimeError('Variable "date4" does not exist.', 20, $this->source); })()), 20, $this->source), "d/m/Y"), "html", null, true);
+        yield "
+
+";
+        // line 22
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date5"]) || array_key_exists("date5", $context) ? $context["date5"] : (function () { throw new RuntimeError('Variable "date5" does not exist.', 22, $this->source); })()), 22, $this->source)), "html", null, true);
+        yield "
+";
+        // line 23
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date5"]) || array_key_exists("date5", $context) ? $context["date5"] : (function () { throw new RuntimeError('Variable "date5" does not exist.', 23, $this->source); })()), 23, $this->source), "d/m/Y"), "html", null, true);
+        yield "
+
+";
+        // line 25
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date6"]) || array_key_exists("date6", $context) ? $context["date6"] : (function () { throw new RuntimeError('Variable "date6" does not exist.', 25, $this->source); })()), 25, $this->source), "d/m/Y H:i:s P", "Europe/Paris"), "html", null, true);
+        yield "
+";
+        // line 26
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date6"]) || array_key_exists("date6", $context) ? $context["date6"] : (function () { throw new RuntimeError('Variable "date6" does not exist.', 26, $this->source); })()), 26, $this->source), "d/m/Y H:i:s P", "Asia/Hong_Kong"), "html", null, true);
+        yield "
+";
+        // line 27
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date6"]) || array_key_exists("date6", $context) ? $context["date6"] : (function () { throw new RuntimeError('Variable "date6" does not exist.', 27, $this->source); })()), 27, $this->source), "d/m/Y H:i:s P", false), "html", null, true);
+        yield "
+";
+        // line 28
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date6"]) || array_key_exists("date6", $context) ? $context["date6"] : (function () { throw new RuntimeError('Variable "date6" does not exist.', 28, $this->source); })()), 28, $this->source), "e", "Europe/Paris"), "html", null, true);
+        yield "
+";
+        // line 29
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date6"]) || array_key_exists("date6", $context) ? $context["date6"] : (function () { throw new RuntimeError('Variable "date6" does not exist.', 29, $this->source); })()), 29, $this->source), "e", false), "html", null, true);
+        yield "
+
+";
+        // line 31
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date7"]) || array_key_exists("date7", $context) ? $context["date7"] : (function () { throw new RuntimeError('Variable "date7" does not exist.', 31, $this->source); })()), 31, $this->source)), "html", null, true);
+        yield "
+";
+        // line 32
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date7"]) || array_key_exists("date7", $context) ? $context["date7"] : (function () { throw new RuntimeError('Variable "date7" does not exist.', 32, $this->source); })()), 32, $this->source), null, "Europe/Paris"), "html", null, true);
+        yield "
+";
+        // line 33
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date7"]) || array_key_exists("date7", $context) ? $context["date7"] : (function () { throw new RuntimeError('Variable "date7" does not exist.', 33, $this->source); })()), 33, $this->source), null, "Asia/Hong_Kong"), "html", null, true);
+        yield "
+";
+        // line 34
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date7"]) || array_key_exists("date7", $context) ? $context["date7"] : (function () { throw new RuntimeError('Variable "date7" does not exist.', 34, $this->source); })()), 34, $this->source), null, false), "html", null, true);
+        yield "
+";
+        // line 35
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, $this->sandbox->ensureToStringAllowed((isset($context["date7"]) || array_key_exists("date7", $context) ? $context["date7"] : (function () { throw new RuntimeError('Variable "date7" does not exist.', 35, $this->source); })()), 35, $this->source), null, "Indian/Mauritius"), "html", null, true);
+        yield "
+
+";
+        // line 37
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, "2010-01-28 15:00:00", null, "Europe/Paris"), "html", null, true);
+        yield "
+";
+        // line 38
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::dateFormatFilter($this->env, "2010-01-28 15:00:00", null, "Asia/Hong_Kong"), "html", null, true);
+        return; yield '';
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function getTemplateName()
+    {
+        return "index.twig";
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function isTraitable()
+    {
+        return false;
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function getDebugInfo()
+    {
+        return array (  166 => 38,  162 => 37,  157 => 35,  153 => 34,  149 => 33,  145 => 32,  141 => 31,  136 => 29,  132 => 28,  128 => 27,  124 => 26,  120 => 25,  115 => 23,  111 => 22,  106 => 20,  102 => 19,  97 => 17,  93 => 16,  88 => 14,  84 => 13,  80 => 12,  76 => 11,  72 => 10,  67 => 8,  63 => 7,  59 => 6,  55 => 5,  51 => 4,  47 => 3,  43 => 2,  40 => 1,);
+    }
+
+    public function getSourceContext()
+    {
+        return new Source("", "index.twig", "");
+    }
+
+    public function checkSecurity()
+    {
+        static $tags = array();
+        static $filters = array("escape" => 2, "date" => 2);
+        static $functions = array();
+
+        try {
+            $this->sandbox->checkSecurity(
+                [],
+                ['escape', 'date'],
+                [],
+                $this->source
+            );
+        } catch (SecurityError $e) {
+            $e->setSourceContext($this->source);
+
+            if ($e instanceof SecurityNotAllowedTagError && isset($tags[$e->getTagName()])) {
+                $e->setTemplateLine($tags[$e->getTagName()]);
+            } elseif ($e instanceof SecurityNotAllowedFilterError && isset($filters[$e->getFilterName()])) {
+                $e->setTemplateLine($filters[$e->getFilterName()]);
+            } elseif ($e instanceof SecurityNotAllowedFunctionError && isset($functions[$e->getFunctionName()])) {
+                $e->setTemplateLine($functions[$e->getFunctionName()]);
+            }
+
+            throw $e;
+        }
+
+    }
+}
+...  366 / 1681 ( 21%)
+...................FCompiled templates that failed on case 1:
+Template: index.twig
+<?php
+
+use Twig\Environment;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
+use Twig\Extension\CoreExtension;
+use Twig\Extension\SandboxExtension;
+use Twig\Markup;
+use Twig\Sandbox\SecurityError;
+use Twig\Sandbox\SecurityNotAllowedTagError;
+use Twig\Sandbox\SecurityNotAllowedFilterError;
+use Twig\Sandbox\SecurityNotAllowedFunctionError;
+use Twig\Source;
+use Twig\Template;
+
+/* index.twig */
+class __TwigTemplate_@HASH@_7cc10555ffcf5a7806aa426ba315656a extends Template
+{
+    private $source;
+    private $macros = [];
+
+    public function __construct(Environment $env)
+    {
+        parent::__construct($env);
+
+        $this->source = $this->getSourceContext();
+
+        $this->parent = false;
+
+        $this->blocks = [
+        ];
+        $this->sandbox = $this->env->getExtension('\Twig\Extension\SandboxExtension');
+        $this->checkSecurity();
+    }
+
+    protected function doDisplay(array $context, array $blocks = [])
+    {
+        $macros = $this->macros;
+        // line 1
+        yield "
+";
+        // line 2
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::convertEncoding(Twig\Extension\CoreExtension::convertEncoding("愛していますか？", "ISO-2022-JP", "UTF-8"), "UTF-8", "ISO-2022-JP"), "html", null, true);
+        yield "
+*";
+        // line 3
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::convertEncoding(Twig\Extension\CoreExtension::convertEncoding("", "ISO-2022-JP", "UTF-8"), "UTF-8", "ISO-2022-JP"), "html", null, true);
+        yield "*
+*";
+        // line 4
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::convertEncoding(Twig\Extension\CoreExtension::convertEncoding(null, "ISO-2022-JP", "UTF-8"), "UTF-8", "ISO-2022-JP"), "html", null, true);
+        yield "*";
+        return; yield '';
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function getTemplateName()
+    {
+        return "index.twig";
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function isTraitable()
+    {
+        return false;
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function getDebugInfo()
+    {
+        return array (  51 => 4,  47 => 3,  43 => 2,  40 => 1,);
+    }
+
+    public function getSourceContext()
+    {
+        return new Source("", "index.twig", "");
+    }
+
+    public function checkSecurity()
+    {
+        static $tags = array();
+        static $filters = array("escape" => 2, "convert_encoding" => 2);
+        static $functions = array();
+
+        try {
+            $this->sandbox->checkSecurity(
+                [],
+                ['escape', 'convert_encoding'],
+                [],
+                $this->source
+            );
+        } catch (SecurityError $e) {
+            $e->setSourceContext($this->source);
+
+            if ($e instanceof SecurityNotAllowedTagError && isset($tags[$e->getTagName()])) {
+                $e->setTemplateLine($tags[$e->getTagName()]);
+            } elseif ($e instanceof SecurityNotAllowedFilterError && isset($filters[$e->getFilterName()])) {
+                $e->setTemplateLine($filters[$e->getFilterName()]);
+            } elseif ($e instanceof SecurityNotAllowedFunctionError && isset($functions[$e->getFunctionName()])) {
+                $e->setTemplateLine($functions[$e->getFunctionName()]);
+            }
+
+            throw $e;
+        }
+
+    }
+}
+........S................................  427 / 1681 ( 25%)
 .............................................................  488 / 1681 ( 29%)
 .............................................................  549 / 1681 ( 32%)
 .............................................................  610 / 1681 ( 36%)
@@ -34,13 +861,141 @@
 
 Time: @TIME@
 
-OK, but incomplete, skipped, or risky tests!
-Tests: 1681, Assertions: 4401, Skipped: 7.
+There were 6 failures:
+
+1) Twig\Tests\IntegrationTest::testIntegration with data set #5 ('functions/date_namedargs.test', '"date" function', '', array('\n{{ date(date, "America/New_...se) }}'), false, array(array('--DATA--\ndate_default_timezo...-04:00', '\ndate_default_timezone_set('...10)]\n', '', '\n04/10/2010 09:45:00 -04:00\...-04:00')), '')
+"date" function (in functions/date_namedargs.test)
+Failed asserting that two strings are equal.
+--- Expected
++++ Actual
+@@ @@
+-'04/10/2010 09:45:00 -04:00\n
+-04/10/2010 09:45:00 -04:00'
++'Twig\Error\Error: Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("DateTimeZone::__construct(): Unknown or bad timezone (America/New_York)") in "index.twig" at line 2.'
+
+@ROOT@/src/Test/IntegrationTestCase.php:253
+@ROOT@/src/Test/IntegrationTestCase.php:82
+
+2) Twig\Tests\IntegrationTest::testIntegration with data set #56 ('filters/date_namedargs.test', '"date" filter', '', array('\n{{ date|date(format='d/m/Y ...o') }}'), false, array(array('--DATA--\ndate_default_timezo...-05:00', '\ndate_default_timezone_set('...10)]\n', '', '\n04/10/2010 08:45:00 -05:00\...-05:00')), '')
+"date" filter (in filters/date_namedargs.test)
+Failed asserting that two strings are equal.
+--- Expected
++++ Actual
+@@ @@
+-'04/10/2010 08:45:00 -05:00\n
+-04/10/2010 08:45:00 -05:00\n
+-04/10/2010 08:45:00 -05:00'
++'Twig\Error\Error: Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("DateTimeZone::__construct(): Unknown or bad timezone (America/Chicago)") in "index.twig" at line 2.'
+
+@ROOT@/src/Test/IntegrationTestCase.php:253
+@ROOT@/src/Test/IntegrationTestCase.php:82
+
+3) Twig\Tests\IntegrationTest::testIntegration with data set #69 ('filters/date_interval.test', '"date" filter (interval support)', '', array('\n{{ date1|date }}\n{{ date1|...e1) }}'), false, array(array('--DATA--\ndate_default_timezo... hours', '\ndate_default_timezone_set('...,\n]\n', '', '\n2 days\n2 days 0 hours\n2 d... hours')), '')
+"date" filter (interval support) (in filters/date_interval.test)
+Failed asserting that two strings are equal.
+--- Expected
++++ Actual
+@@ @@
+-'2 days\n
+-2 days 0 hours\n
+-2 days 0 hours'
++'Twig\Error\Error: DateInvalidTimeZoneException: DateTimeZone::__construct(): Unknown or bad timezone (America/New_York)'
+
+@ROOT@/src/Test/IntegrationTestCase.php:253
+@ROOT@/src/Test/IntegrationTestCase.php:82
+
+4) Twig\Tests\IntegrationTest::testIntegration with data set #70 ('filters/date_immutable.test', '"date" filter', '', array('\n{{ date1|date }}\n{{ date1|...se) }}'), false, array(array('--DATA--\ndate_default_timezo...w_York', '\ndate_default_timezone_set('...,\n]\n', '', '\nOctober 4, 2010 13:45\n04/1...w_York')), '')
+"date" filter (in filters/date_immutable.test)
+Failed asserting that two strings are equal.
+--- Expected
++++ Actual
+@@ @@
+-'October 4, 2010 13:45\n
+-04/10/2010\n
+-04/10/2010 19:45:00\n
+-04/10/2010 07:45:00\n
+-04/10/2010 13:45:00\n
+-04/10/2010 14:45:00\n
+-\n
+-04/10/2010 19:45:00 +02:00\n
+-05/10/2010 01:45:00 +08:00\n
+-04/10/2010 13:45:00 -04:00\n
+-Europe/Paris\n
+-America/New_York'
++'Twig\Error\Error: PHPUnit\Framework\Error\Notice: date_default_timezone_set(): Timezone ID 'Europe/Paris' is invalid'
+
+@ROOT@/src/Test/IntegrationTestCase.php:253
+@ROOT@/src/Test/IntegrationTestCase.php:82
+
+5) Twig\Tests\IntegrationTest::testIntegration with data set #74 ('filters/date.test', '"date" filter', '', array('\n{{ date1|date }}\n{{ date1|...g") }}'), false, array(array('--DATA--\ndate_default_timezo... 22:00', '\ndate_default_timezone_set('...,\n]\n', '', '\nOctober 4, 2010 13:45\n04/1... 22:00')), '')
+"date" filter (in filters/date.test)
+Failed asserting that two strings are equal.
+--- Expected
++++ Actual
+@@ @@
+-'October 4, 2010 13:45\n
+-04/10/2010\n
+-04/10/2010 19:45:00\n
+-04/10/2010 19:45:00 +08:00\n
+-04/10/2010 06:45:00 -05:00\n
+-Europe/Paris\n
+-04/10/2010 13:45:00\n
+-\n
+-October 4, 2010 13:45\n
+-04/10/2010\n
+-04/10/2010 19:45:00\n
+-04/10/2010 07:45:00\n
+-04/10/2010 13:45:00\n
+-\n
+-October 4, 2010 13:45\n
+-04/10/2010\n
+-\n
+-October 4, 2010 15:45\n
+-04/10/2010\n
+-\n
+-January 2, 1964 04:04\n
+-02/01/1964\n
+-\n
+-04/10/2010 19:45:00 +02:00\n
+-05/10/2010 01:45:00 +08:00\n
+-04/10/2010 13:45:00 -04:00\n
+-Europe/Paris\n
+-America/New_York\n
+-\n
+-January 28, 2010 12:00\n
+-January 28, 2010 12:00\n
+-January 28, 2010 19:00\n
+-January 28, 2010 15:00\n
+-January 28, 2010 15:00\n
+-\n
+-January 28, 2010 15:00\n
+-January 28, 2010 22:00'
++'Twig\Error\Error: PHPUnit\Framework\Error\Notice: date_default_timezone_set(): Timezone ID 'Europe/Paris' is invalid'
+
+@ROOT@/src/Test/IntegrationTestCase.php:253
+@ROOT@/src/Test/IntegrationTestCase.php:82
+
+6) Twig\Tests\IntegrationTest::testIntegration with data set #97 ('filters/convert_encoding.test', '"convert_encoding" filter', '', array('\n{{ "愛していますか？"|convert_encod...') }}*'), false, array(array('--DATA--\nreturn []\n--EXPECT...**\n**', '\nreturn []\n', '', '\n愛していますか？\n**\n**')), '')
+"convert_encoding" filter (in filters/convert_encoding.test)
+Failed asserting that two strings are equal.
+--- Expected
++++ Actual
+@@ @@
+-'愛していますか？\n
+-**\n
+-**'
++'Twig\Error\Error: Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("iconv(): Wrong encoding, conversion from "UTF-8" to "ISO-2022-JP" is not allowed") in "index.twig" at line 2.'
+
+@ROOT@/src/Test/IntegrationTestCase.php:253
+@ROOT@/src/Test/IntegrationTestCase.php:82
+
+FAILURES!
+Tests: 1681, Assertions: 4404, Failures: 6, Skipped: 5.
 
-Remaining self deprecation notices (463)
+Remaining self deprecation notices (465)
 
-  442x: Method ReflectionProperty::setAccessible() is deprecated since 8.5, as it has no effect since PHP 8.1
-    367x in IntegrationTest::testIntegration from Twig\Tests
+  444x: Method ReflectionProperty::setAccessible() is deprecated since 8.5, as it has no effect since PHP 8.1
+    369x in IntegrationTest::testIntegration from Twig\Tests
     22x in BlackfireTest::testDump from Twig\Tests\Profiler\Dumper
     22x in HtmlTest::testDump from Twig\Tests\Profiler\Dumper
     22x in TextTest::testDump from Twig\Tests\Profiler\Dumper
