--- php
+++ phl
@@ -1,5 +1,3 @@
-Xdebug: [Config] The setting 'xdebug.overload_var_dump' has been removed, see the upgrading guide at https://xdebug.org/docs/upgrade_guide#changed-xdebug.overload_var_dump (See: https://xdebug.org/docs/errors#CFG-C-REMOVED)
-Xdebug: [Config] The setting 'xdebug.overload_var_dump' has been removed, see the upgrading guide at https://xdebug.org/docs/upgrade_guide#changed-xdebug.overload_var_dump (See: https://xdebug.org/docs/errors#CFG-C-REMOVED)
 PHPUnit 9.6.36 by Sebastian Bergmann and contributors.
 
 Testing
@@ -8,8 +6,123 @@
 .............................................................  183 / 1681 ( 10%)
 .............................................................  244 / 1681 ( 14%)
 ...............................................S.............  305 / 1681 ( 18%)
-.....................S.S.....................................  366 / 1681 ( 21%)
-............................S................................  427 / 1681 ( 25%)
+.............................................................  366 / 1681 ( 21%)
+...................FCompiled templates that failed on case 1:
+Template: index.twig
+<?php
+
+use Twig\Environment;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
+use Twig\Extension\CoreExtension;
+use Twig\Extension\SandboxExtension;
+use Twig\Markup;
+use Twig\Sandbox\SecurityError;
+use Twig\Sandbox\SecurityNotAllowedTagError;
+use Twig\Sandbox\SecurityNotAllowedFilterError;
+use Twig\Sandbox\SecurityNotAllowedFunctionError;
+use Twig\Source;
+use Twig\Template;
+
+/* index.twig */
+class __TwigTemplate_@HASH@_7cc10555ffcf5a7806aa426ba315656a extends Template
+{
+    private $source;
+    private $macros = [];
+
+    public function __construct(Environment $env)
+    {
+        parent::__construct($env);
+
+        $this->source = $this->getSourceContext();
+
+        $this->parent = false;
+
+        $this->blocks = [
+        ];
+        $this->sandbox = $this->env->getExtension('\Twig\Extension\SandboxExtension');
+        $this->checkSecurity();
+    }
+
+    protected function doDisplay(array $context, array $blocks = [])
+    {
+        $macros = $this->macros;
+        // line 1
+        yield "
+";
+        // line 2
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::convertEncoding(Twig\Extension\CoreExtension::convertEncoding("愛していますか？", "ISO-2022-JP", "UTF-8"), "UTF-8", "ISO-2022-JP"), "html", null, true);
+        yield "
+*";
+        // line 3
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::convertEncoding(Twig\Extension\CoreExtension::convertEncoding("", "ISO-2022-JP", "UTF-8"), "UTF-8", "ISO-2022-JP"), "html", null, true);
+        yield "*
+*";
+        // line 4
+        yield Twig\Extension\EscaperExtension::escape($this->env, Twig\Extension\CoreExtension::convertEncoding(Twig\Extension\CoreExtension::convertEncoding(null, "ISO-2022-JP", "UTF-8"), "UTF-8", "ISO-2022-JP"), "html", null, true);
+        yield "*";
+        return; yield '';
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function getTemplateName()
+    {
+        return "index.twig";
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function isTraitable()
+    {
+        return false;
+    }
+
+    /**
+     * @codeCoverageIgnore
+     */
+    public function getDebugInfo()
+    {
+        return array (  51 => 4,  47 => 3,  43 => 2,  40 => 1,);
+    }
+
+    public function getSourceContext()
+    {
+        return new Source("", "index.twig", "");
+    }
+
+    public function checkSecurity()
+    {
+        static $tags = array();
+        static $filters = array("escape" => 2, "convert_encoding" => 2);
+        static $functions = array();
+
+        try {
+            $this->sandbox->checkSecurity(
+                [],
+                ['escape', 'convert_encoding'],
+                [],
+                $this->source
+            );
+        } catch (SecurityError $e) {
+            $e->setSourceContext($this->source);
+
+            if ($e instanceof SecurityNotAllowedTagError && isset($tags[$e->getTagName()])) {
+                $e->setTemplateLine($tags[$e->getTagName()]);
+            } elseif ($e instanceof SecurityNotAllowedFilterError && isset($filters[$e->getFilterName()])) {
+                $e->setTemplateLine($filters[$e->getFilterName()]);
+            } elseif ($e instanceof SecurityNotAllowedFunctionError && isset($functions[$e->getFunctionName()])) {
+                $e->setTemplateLine($functions[$e->getFunctionName()]);
+            }
+
+            throw $e;
+        }
+
+    }
+}
+........S................................  427 / 1681 ( 25%)
 .............................................................  488 / 1681 ( 29%)
 .............................................................  549 / 1681 ( 32%)
 .............................................................  610 / 1681 ( 36%)
@@ -34,13 +147,29 @@
 
 Time: @TIME@
 
-OK, but incomplete, skipped, or risky tests!
-Tests: 1681, Assertions: 4401, Skipped: 7.
+There was 1 failure:
+
+1) Twig\Tests\IntegrationTest::testIntegration with data set #97 ('filters/convert_encoding.test', '"convert_encoding" filter', '', array('\n{{ "愛していますか？"|convert_encod...') }}*'), false, array(array('--DATA--\nreturn []\n--EXPECT...**\n**', '\nreturn []\n', '', '\n愛していますか？\n**\n**')), '')
+"convert_encoding" filter (in filters/convert_encoding.test)
+Failed asserting that two strings are equal.
+--- Expected
++++ Actual
+@@ @@
+-'愛していますか？\n
+-**\n
+-**'
++'Twig\Error\Error: Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("iconv(): Wrong encoding, conversion from "UTF-8" to "ISO-2022-JP" is not allowed") in "index.twig" at line 2.'
+
+@ROOT@/src/Test/IntegrationTestCase.php:253
+@ROOT@/src/Test/IntegrationTestCase.php:82
+
+FAILURES!
+Tests: 1681, Assertions: 4404, Failures: 1, Skipped: 5.
 
-Remaining self deprecation notices (463)
+Remaining self deprecation notices (465)
 
-  442x: Method ReflectionProperty::setAccessible() is deprecated since 8.5, as it has no effect since PHP 8.1
-    367x in IntegrationTest::testIntegration from Twig\Tests
+  444x: Method ReflectionProperty::setAccessible() is deprecated since 8.5, as it has no effect since PHP 8.1
+    369x in IntegrationTest::testIntegration from Twig\Tests
     22x in BlackfireTest::testDump from Twig\Tests\Profiler\Dumper
     22x in HtmlTest::testDump from Twig\Tests\Profiler\Dumper
     22x in TextTest::testDump from Twig\Tests\Profiler\Dumper
