--- php
+++ phl
@@ -1,2 +1,14 @@
-No changes needed.
-@exit@ 0
+--- a/docs/validators/Charset.md
++++ b/docs/validators/Charset.md
+@@ -20,7 +20,7 @@
+ // → "açúcar" must consist only of characters from the "ASCII" character-set
+
+ v::charset('ISO-8859-1', 'EUC-JP')->assert('日本国');
+-// Validation passes successfully
++// 𝙭 Could not instantiate "charset": Invalid charset provided: "EUC-JP"
+ ```
+
+ The array format is a logic OR, not AND.
+
+Changes needed in 1 files.
+@exit@ 1
