--- php
+++ phl
@@ -2,7 +2,7 @@
   ............................................................................
   ............................................................................
   ............................................................................
-  ............................................................................
+  ..........................................................⨯.................
   ............................................................................
   ............................................................................
   ............................................................................
@@ -15,8 +15,28 @@
   ............................................................................
   ............................................................................
   .....................
+  ────────────────────────────────────────────────────────────────────────────
+   FAILED  Tests\feature\ValidationExceptionStackTraceTest > Should go not o…
+  Failed asserting that two strings are identical.
+  -'@ROOT@/tests/feature/ValidationExceptionStackTraceTest.php'
+  +'@ROOT@/vendor/pestphp/pest/src/Factories/TestCaseFactory.php(178) : eval()'d code'
+
+
+  at tests/feature/ValidationExceptionStackTraceTest.php:81
+     77▕         $trace[] = __FILE__;
+     78▕
+     79▕         throw new ValidationException('message', $resultQuery, ...$trace);
+     80▕     } catch (ValidationException $e) {
+  ➜  81▕         expect($e->getFile())->toBe(__FILE__);
+     82▕         expect($e->getLine())->toBe(__LINE__ - 3);
+     83▕     }
+     84▕ });
+     85▕
+
+  1   tests/feature/ValidationExceptionStackTraceTest.php:81
+
 
-  Tests:    1161 passed (1937 assertions)
+  Tests:    1 failed, 1160 passed (1936 assertions)
   Duration: @TIME@
 
-@exit@ 0
+@exit@ 1
