--- php
+++ phl
@@ -480,208 +480,98 @@
 @ROOT@/tests/src/Stubs/WithUninitialized.php
 @ROOT@/tests/src/Stubs/WithUnionTypeNested.php
 @ROOT@/tests/src/TestCase.php
-@ROOT@/tests/src/Transformers/StubTransformer.php
-@ROOT@/tests/src/Validators/Core/ConcreteComparison.php
-@ROOT@/tests/src/Validators/Core/ConcreteEnvelope.php
-@ROOT@/tests/src/Validators/Core/ConcreteFilteredArray.php
-@ROOT@/tests/src/Validators/Core/ConcreteFilteredString.php
-@ROOT@/tests/src/Validators/Core/ConcreteLogicalComposite.php
-@ROOT@/tests/src/Validators/Core/ConcreteSimple.php
-@ROOT@/tests/src/Validators/CustomRule.php
-@ROOT@/tests/src/Validators/Invalid.php
-@ROOT@/tests/src/Validators/MyAbstractClass.php
-@ROOT@/tests/src/Validators/NoConstructor.php
-@ROOT@/tests/src/Validators/NonPublic.php
-@ROOT@/tests/src/Validators/Stub.php
-@ROOT@/tests/src/Validators/Valid.php
-@ROOT@/tests/src/Validators/WithDependency.php
-@ROOT@/tests/unit/AutowiringLookupTest.php
-@ROOT@/tests/unit/ContainerRegistryTest.php
-@ROOT@/tests/unit/Exceptions/InvalidValidatorExceptionTest.php
-@ROOT@/tests/unit/Exceptions/MissingComposerDependencyExceptionTest.php
-@ROOT@/tests/unit/FluentValidatorFactoryTest.php
-@ROOT@/tests/unit/Helpers/CanValidateDateTimeTest.php
-@ROOT@/tests/unit/Helpers/CanValidateUndefinedTest.php
-@ROOT@/tests/unit/Helpers/DataLoaderTest.php
-@ROOT@/tests/unit/IdTest.php
-@ROOT@/tests/unit/Message/Formatter/FirstResultStringFormatterTest.php
-@ROOT@/tests/unit/Message/Formatter/NestedArrayFormatterTest.php
-@ROOT@/tests/unit/Message/Formatter/NestedListStringFormatterTest.php
-@ROOT@/tests/unit/Message/Formatter/OnlyFailedChildrenResultFilterTest.php
-@ROOT@/tests/unit/Message/Formatter/TemplateResolverTest.php
-@ROOT@/tests/unit/Message/InterpolationRendererTest.php
-@ROOT@/tests/unit/Message/Parameters/ResultHandlerTest.php
-@ROOT@/tests/unit/Message/StandardFormatter/ResultCreator.php
-@ROOT@/tests/unit/Message/TemplateRegistryTest.php
-@ROOT@/tests/unit/ResultQueryTest.php
-@ROOT@/tests/unit/ResultTest.php
-@ROOT@/tests/unit/Transformers/PrefixTest.php
-@ROOT@/tests/unit/ValidatorBuilderTest.php
-@ROOT@/tests/unit/Validators/AfterTest.php
-@ROOT@/tests/unit/Validators/AllOfTest.php
-@ROOT@/tests/unit/Validators/AllTest.php
-@ROOT@/tests/unit/Validators/AlnumTest.php
-@ROOT@/tests/unit/Validators/AlphaTest.php
-@ROOT@/tests/unit/Validators/AlwaysInvalidTest.php
-@ROOT@/tests/unit/Validators/AlwaysValidTest.php
-@ROOT@/tests/unit/Validators/AnyOfTest.php
-@ROOT@/tests/unit/Validators/ArrayTypeTest.php
-@ROOT@/tests/unit/Validators/ArrayValTest.php
-@ROOT@/tests/unit/Validators/AttributesTest.php
-@ROOT@/tests/unit/Validators/Base64Test.php
-@ROOT@/tests/unit/Validators/BaseTest.php
-@ROOT@/tests/unit/Validators/BetweenExclusiveTest.php
-@ROOT@/tests/unit/Validators/BetweenTest.php
-@ROOT@/tests/unit/Validators/BlankTest.php
-@ROOT@/tests/unit/Validators/BoolTypeTest.php
-@ROOT@/tests/unit/Validators/BoolValTest.php
-@ROOT@/tests/unit/Validators/BsnTest.php
-@ROOT@/tests/unit/Validators/CallableTypeTest.php
-@ROOT@/tests/unit/Validators/CharsetTest.php
-@ROOT@/tests/unit/Validators/CnhTest.php
-@ROOT@/tests/unit/Validators/CnpjTest.php
-@ROOT@/tests/unit/Validators/ConsonantTest.php
-@ROOT@/tests/unit/Validators/ContainsAnyTest.php
-@ROOT@/tests/unit/Validators/ContainsCountTest.php
-@ROOT@/tests/unit/Validators/ContainsTest.php
-@ROOT@/tests/unit/Validators/ControlTest.php
-@ROOT@/tests/unit/Validators/Core/ComparisonTest.php
-@ROOT@/tests/unit/Validators/Core/CompositeTest.php
-@ROOT@/tests/unit/Validators/Core/EnvelopeTest.php
-@ROOT@/tests/unit/Validators/Core/FilteredArrayTest.php
-@ROOT@/tests/unit/Validators/Core/FilteredStringTest.php
-@ROOT@/tests/unit/Validators/Core/ReducerTest.php
-@ROOT@/tests/unit/Validators/Core/SimpleTest.php
-@ROOT@/tests/unit/Validators/CountableTest.php
-@ROOT@/tests/unit/Validators/CountryCodeTest.php
-@ROOT@/tests/unit/Validators/CpfTest.php
-@ROOT@/tests/unit/Validators/CreditCardTest.php
-@ROOT@/tests/unit/Validators/CurrencyCodeTest.php
-@ROOT@/tests/unit/Validators/DateTest.php
-@ROOT@/tests/unit/Validators/DateTimeDiffTest.php
-@ROOT@/tests/unit/Validators/DateTimeTest.php
-@ROOT@/tests/unit/Validators/DecimalTest.php
-@ROOT@/tests/unit/Validators/DigitTest.php
-@ROOT@/tests/unit/Validators/DirectoryTest.php
-@ROOT@/tests/unit/Validators/DomainTest.php
-@ROOT@/tests/unit/Validators/EachKeyTest.php
-@ROOT@/tests/unit/Validators/EachTest.php
-@ROOT@/tests/unit/Validators/EmailTest.php
-@ROOT@/tests/unit/Validators/EmojiTest.php
-@ROOT@/tests/unit/Validators/EndsWithTest.php
-@ROOT@/tests/unit/Validators/EqualsTest.php
-@ROOT@/tests/unit/Validators/EquivalentTest.php
-@ROOT@/tests/unit/Validators/EvenTest.php
-@ROOT@/tests/unit/Validators/ExecutableTest.php
-@ROOT@/tests/unit/Validators/ExistsTest.php
-@ROOT@/tests/unit/Validators/ExtensionTest.php
-@ROOT@/tests/unit/Validators/FactorTest.php
-@ROOT@/tests/unit/Validators/FactoryTest.php
-@ROOT@/tests/unit/Validators/FalseValTest.php
-@ROOT@/tests/unit/Validators/FalsyTest.php
-@ROOT@/tests/unit/Validators/FileTest.php
-@ROOT@/tests/unit/Validators/FiniteTest.php
-@ROOT@/tests/unit/Validators/FloatTypeTest.php
-@ROOT@/tests/unit/Validators/FloatValTest.php
-@ROOT@/tests/unit/Validators/FormatTest.php
-@ROOT@/tests/unit/Validators/FormattedTest.php
-@ROOT@/tests/unit/Validators/GivenTest.php
-@ROOT@/tests/unit/Validators/GraphTest.php
-@ROOT@/tests/unit/Validators/GreaterThanOrEqualTest.php
-@ROOT@/tests/unit/Validators/GreaterThanTest.php
-@ROOT@/tests/unit/Validators/HetuTest.php
-@ROOT@/tests/unit/Validators/HexRgbColorTest.php
-@ROOT@/tests/unit/Validators/IbanTest.php
-@ROOT@/tests/unit/Validators/IdenticalTest.php
-@ROOT@/tests/unit/Validators/ImageTest.php
-@ROOT@/tests/unit/Validators/ImeiTest.php
-@ROOT@/tests/unit/Validators/InTest.php
-@ROOT@/tests/unit/Validators/InfiniteTest.php
-@ROOT@/tests/unit/Validators/InstanceTest.php
-@ROOT@/tests/unit/Validators/IntTypeTest.php
-@ROOT@/tests/unit/Validators/IntValTest.php
-@ROOT@/tests/unit/Validators/IpTest.php
-@ROOT@/tests/unit/Validators/IsbnTest.php
-@ROOT@/tests/unit/Validators/IterableTypeTest.php
-@ROOT@/tests/unit/Validators/IterableValTest.php
-@ROOT@/tests/unit/Validators/JsonTest.php
-@ROOT@/tests/unit/Validators/KeyExistsTest.php
-@ROOT@/tests/unit/Validators/KeyOptionalTest.php
-@ROOT@/tests/unit/Validators/KeySetTest.php
-@ROOT@/tests/unit/Validators/KeyTest.php
-@ROOT@/tests/unit/Validators/LanguageCodeTest.php
-@ROOT@/tests/unit/Validators/LeapDateTest.php
-@ROOT@/tests/unit/Validators/LeapYearTest.php
-@ROOT@/tests/unit/Validators/LengthTest.php
-@ROOT@/tests/unit/Validators/LessThanOrEqualTest.php
-@ROOT@/tests/unit/Validators/LessThanTest.php
-@ROOT@/tests/unit/Validators/LowercaseTest.php
-@ROOT@/tests/unit/Validators/LuhnTest.php
-@ROOT@/tests/unit/Validators/MacAddressTest.php
-@ROOT@/tests/unit/Validators/MaxTest.php
-@ROOT@/tests/unit/Validators/MimetypeTest.php
-@ROOT@/tests/unit/Validators/MinTest.php
-@ROOT@/tests/unit/Validators/MultipleTest.php
-@ROOT@/tests/unit/Validators/NegativeTest.php
-@ROOT@/tests/unit/Validators/NfeAccessKeyTest.php
-@ROOT@/tests/unit/Validators/NifTest.php
-@ROOT@/tests/unit/Validators/NipTest.php
-@ROOT@/tests/unit/Validators/NoneOfTest.php
-@ROOT@/tests/unit/Validators/NotTest.php
-@ROOT@/tests/unit/Validators/NullOrTest.php
-@ROOT@/tests/unit/Validators/NullTypeTest.php
-@ROOT@/tests/unit/Validators/NumberTest.php
-@ROOT@/tests/unit/Validators/NumericValTest.php
-@ROOT@/tests/unit/Validators/ObjectTypeTest.php
-@ROOT@/tests/unit/Validators/OddTest.php
-@ROOT@/tests/unit/Validators/OneOfTest.php
-@ROOT@/tests/unit/Validators/PeselTest.php
-@ROOT@/tests/unit/Validators/PhoneTest.php
-@ROOT@/tests/unit/Validators/PisTest.php
-@ROOT@/tests/unit/Validators/PolishIdCardTest.php
-@ROOT@/tests/unit/Validators/PortugueseNifTest.php
-@ROOT@/tests/unit/Validators/PositiveTest.php
-@ROOT@/tests/unit/Validators/PostalCodeTest.php
-@ROOT@/tests/unit/Validators/PrintableTest.php
-@ROOT@/tests/unit/Validators/PropertyExistsTest.php
-@ROOT@/tests/unit/Validators/PropertyOptionalTest.php
-@ROOT@/tests/unit/Validators/PropertyTest.php
-@ROOT@/tests/unit/Validators/PublicDomainSuffixTest.php
-@ROOT@/tests/unit/Validators/PunctTest.php
-@ROOT@/tests/unit/Validators/ReadableTest.php
-@ROOT@/tests/unit/Validators/RegexTest.php
-@ROOT@/tests/unit/Validators/ResourceTypeTest.php
-@ROOT@/tests/unit/Validators/RomanTest.php
-@ROOT@/tests/unit/Validators/SatisfiesTest.php
-@ROOT@/tests/unit/Validators/ScalarValTest.php
-@ROOT@/tests/unit/Validators/ShortCircuitTest.php
-@ROOT@/tests/unit/Validators/SizeTest.php
-@ROOT@/tests/unit/Validators/SlugTest.php
-@ROOT@/tests/unit/Validators/SortedTest.php
-@ROOT@/tests/unit/Validators/SpaceTest.php
-@ROOT@/tests/unit/Validators/SpacedTest.php
-@ROOT@/tests/unit/Validators/StartsWithTest.php
-@ROOT@/tests/unit/Validators/StringTypeTest.php
-@ROOT@/tests/unit/Validators/StringValTest.php
-@ROOT@/tests/unit/Validators/SubdivisionCodeTest.php
-@ROOT@/tests/unit/Validators/SubsetTest.php
-@ROOT@/tests/unit/Validators/SymbolicLinkTest.php
-@ROOT@/tests/unit/Validators/TimeTest.php
-@ROOT@/tests/unit/Validators/TldTest.php
-@ROOT@/tests/unit/Validators/TrimmedTest.php
-@ROOT@/tests/unit/Validators/TrueValTest.php
-@ROOT@/tests/unit/Validators/UndefOrTest.php
-@ROOT@/tests/unit/Validators/UndefTest.php
-@ROOT@/tests/unit/Validators/UniqueTest.php
-@ROOT@/tests/unit/Validators/UppercaseTest.php
-@ROOT@/tests/unit/Validators/UrlTest.php
-@ROOT@/tests/unit/Validators/UuidTest.php
-@ROOT@/tests/unit/Validators/VersionTest.php
-@ROOT@/tests/unit/Validators/VowelTest.php
-@ROOT@/tests/unit/Validators/WhenTest.php
-@ROOT@/tests/unit/Validators/WritableTest.php
-@ROOT@/tests/unit/Validators/XdigitTest.php
+Uncaught PHPStan\BetterReflection\SourceLocator\Ast\Exception\ParseToAstFailure: AST failed to parse in located source (line 3): Syntax error, unexpected T_EMPTY
 
- [OK] No errors
+<?php
 
-@exit@ 0
+function empty()
+{
+}
+ in phar://phpstan.phar/vendor/ondrejmirtes/better-reflection/src/SourceLocator/Ast/Exception/ParseToAstFailure.php:43
+#0 phar://phpstan.phar/vendor/ondrejmirtes/better-reflection/src/SourceLocator/Ast/Locator.php(49): PHPStan\BetterReflection\SourceLocator\Ast\Exception\ParseToAstFailure::fromLocatedSource()
+#1 phar://phpstan.phar/vendor/ondrejmirtes/better-reflection/src/SourceLocator/Ast/Locator.php(33): PHPStan\BetterReflection\SourceLocator\Ast\Locator->findReflectionsOfType()
+#2 phar://phpstan.phar/vendor/ondrejmirtes/better-reflection/src/SourceLocator/Type/AbstractSourceLocator.php(42): PHPStan\BetterReflection\SourceLocator\Ast\Locator->findReflection()
+#3 phar://phpstan.phar/src/Reflection/BetterReflection/SourceLocator/PhpVersionBlacklistSourceLocator.php(35): PHPStan\BetterReflection\SourceLocator\Type\AbstractSourceLocator->locateIdentifier()
+#4 phar://phpstan.phar/vendor/ondrejmirtes/better-reflection/src/SourceLocator/Type/AggregateSourceLocator.php(26): PHPStan\Reflection\BetterReflection\SourceLocator\PhpVersionBlacklistSourceLocator->locateIdentifier()
+#5 phar://phpstan.phar/src/Reflection/BetterReflection/SourceLocator/LazySourceLocator.php(31): PHPStan\BetterReflection\SourceLocator\Type\AggregateSourceLocator->locateIdentifier()
+#6 phar://phpstan.phar/src/Reflection/BetterReflection/Reflector/MemoizingReflector.php(98): PHPStan\Reflection\BetterReflection\SourceLocator\LazySourceLocator->locateIdentifier()
+#7 phar://phpstan.phar/src/Reflection/BetterReflection/BetterReflectionProvider.php(322): PHPStan\Reflection\BetterReflection\Reflector\MemoizingReflector->reflectFunction()
+#8 phar://phpstan.phar/src/Reflection/BetterReflection/BetterReflectionProvider.php(421): {closure:PHPStan\Reflection\BetterReflection\BetterReflectionProvider::resolveFunctionName():320}()
+#9 phar://phpstan.phar/src/Reflection/BetterReflection/BetterReflectionProvider.php(320): PHPStan\Reflection\BetterReflection\BetterReflectionProvider->resolveName()
+#10 phar://phpstan.phar/src/Reflection/BetterReflection/BetterReflectionProvider.php(234): PHPStan\Reflection\BetterReflection\BetterReflectionProvider->resolveFunctionName()
+#11 phar://phpstan.phar/src/Reflection/ReflectionProvider/MemoizingReflectionProvider.php(64): PHPStan\Reflection\BetterReflection\BetterReflectionProvider->hasFunction()
+#12 phar://phpstan.phar/src/Type/Constant/ConstantStringType.php(185): PHPStan\Reflection\ReflectionProvider\MemoizingReflectionProvider->hasFunction()
+#13 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(3006): PHPStan\Type\Constant\ConstantStringType->isCallable()
+#14 phar://phpstan.phar/src/Analyser/ExprHandler/MethodCallHandler.php(136): PHPStan\Analyser\NodeScopeResolver->processArgs()
+#15 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(2216): PHPStan\Analyser\ExprHandler\MethodCallHandler->processExpr()
+#16 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(900): PHPStan\Analyser\NodeScopeResolver->processExprNode()
+#17 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(609): PHPStan\Analyser\NodeScopeResolver->processStmtNode()
+#18 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(583): PHPStan\Analyser\NodeScopeResolver->processStmtNodesInternalWithoutFlushingPendingFibers()
+#19 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(820): PHPStan\Analyser\NodeScopeResolver->processStmtNodesInternal()
+#20 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(609): PHPStan\Analyser\NodeScopeResolver->processStmtNode()
+#21 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(583): PHPStan\Analyser\NodeScopeResolver->processStmtNodesInternalWithoutFlushingPendingFibers()
+#22 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(1034): PHPStan\Analyser\NodeScopeResolver->processStmtNodesInternal()
+#23 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(609): PHPStan\Analyser\NodeScopeResolver->processStmtNode()
+#24 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(583): PHPStan\Analyser\NodeScopeResolver->processStmtNodesInternalWithoutFlushingPendingFibers()
+#25 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(972): PHPStan\Analyser\NodeScopeResolver->processStmtNodesInternal()
+#26 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(458): PHPStan\Analyser\NodeScopeResolver->processStmtNode()
+#27 phar://phpstan.phar/src/Analyser/FileAnalyser.php(122): PHPStan\Analyser\NodeScopeResolver->processNodes()
+#28 phar://phpstan.phar/src/Analyser/Analyser.php(81): PHPStan\Analyser\FileAnalyser->analyseFile()
+#29 phar://phpstan.phar/src/Command/AnalyserRunner.php(70): PHPStan\Analyser\Analyser->analyse()
+#30 phar://phpstan.phar/src/Command/AnalyseApplication.php(188): PHPStan\Command\AnalyserRunner->runAnalyser()
+#31 phar://phpstan.phar/src/Command/AnalyseApplication.php(80): PHPStan\Command\AnalyseApplication->runAnalyser()
+#32 phar://phpstan.phar/src/Command/AnalyseCommand.php(263): PHPStan\Command\AnalyseApplication->analyse()
+#33 phar://phpstan.phar/vendor/symfony/console/Command/Command.php(259): PHPStan\Command\AnalyseCommand->execute()
+#34 phar://phpstan.phar/vendor/symfony/console/Application.php(868): _PHPStan_b3f880679\Symfony\Component\Console\Command\Command->run()
+#35 phar://phpstan.phar/vendor/symfony/console/Application.php(261): _PHPStan_b3f880679\Symfony\Component\Console\Application->doRunCommand()
+#36 phar://phpstan.phar/vendor/symfony/console/Application.php(157): _PHPStan_b3f880679\Symfony\Component\Console\Application->doRun()
+#37 phar://phpstan.phar/bin/phpstan(109): _PHPStan_b3f880679\Symfony\Component\Console\Application->run()
+#38 phar://phpstan.phar/bin/phpstan(110): {closure:phar://phpstan.phar/bin/phpstan:16}()
+#39 @ROOT@/vendor/phpstan/phpstan/phpstan(8): require('...')
+#40 @ROOT@/vendor/bin/phpstan(119): include('...')
+#41 {main}
+
+Caused by:
+Uncaught PhpParser\Error: Syntax error, unexpected T_EMPTY on line 3 in phar://phpstan.phar/src/Parser/PhpParserDecorator.php:33#0 phar://phpstan.phar/vendor/ondrejmirtes/better-reflection/src/SourceLocator/Ast/Locator.php(46): PHPStan\Parser\PhpParserDecorator->parse()
+#1 phar://phpstan.phar/vendor/ondrejmirtes/better-reflection/src/SourceLocator/Ast/Locator.php(33): PHPStan\BetterReflection\SourceLocator\Ast\Locator->findReflectionsOfType()
+#2 phar://phpstan.phar/vendor/ondrejmirtes/better-reflection/src/SourceLocator/Type/AbstractSourceLocator.php(42): PHPStan\BetterReflection\SourceLocator\Ast\Locator->findReflection()
+#3 phar://phpstan.phar/src/Reflection/BetterReflection/SourceLocator/PhpVersionBlacklistSourceLocator.php(35): PHPStan\BetterReflection\SourceLocator\Type\AbstractSourceLocator->locateIdentifier()
+#4 phar://phpstan.phar/vendor/ondrejmirtes/better-reflection/src/SourceLocator/Type/AggregateSourceLocator.php(26): PHPStan\Reflection\BetterReflection\SourceLocator\PhpVersionBlacklistSourceLocator->locateIdentifier()
+#5 phar://phpstan.phar/src/Reflection/BetterReflection/SourceLocator/LazySourceLocator.php(31): PHPStan\BetterReflection\SourceLocator\Type\AggregateSourceLocator->locateIdentifier()
+#6 phar://phpstan.phar/src/Reflection/BetterReflection/Reflector/MemoizingReflector.php(98): PHPStan\Reflection\BetterReflection\SourceLocator\LazySourceLocator->locateIdentifier()
+#7 phar://phpstan.phar/src/Reflection/BetterReflection/BetterReflectionProvider.php(322): PHPStan\Reflection\BetterReflection\Reflector\MemoizingReflector->reflectFunction()
+#8 phar://phpstan.phar/src/Reflection/BetterReflection/BetterReflectionProvider.php(421): {closure:PHPStan\Reflection\BetterReflection\BetterReflectionProvider::resolveFunctionName():320}()
+#9 phar://phpstan.phar/src/Reflection/BetterReflection/BetterReflectionProvider.php(320): PHPStan\Reflection\BetterReflection\BetterReflectionProvider->resolveName()
+#10 phar://phpstan.phar/src/Reflection/BetterReflection/BetterReflectionProvider.php(234): PHPStan\Reflection\BetterReflection\BetterReflectionProvider->resolveFunctionName()
+#11 phar://phpstan.phar/src/Reflection/ReflectionProvider/MemoizingReflectionProvider.php(64): PHPStan\Reflection\BetterReflection\BetterReflectionProvider->hasFunction()
+#12 phar://phpstan.phar/src/Type/Constant/ConstantStringType.php(185): PHPStan\Reflection\ReflectionProvider\MemoizingReflectionProvider->hasFunction()
+#13 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(3006): PHPStan\Type\Constant\ConstantStringType->isCallable()
+#14 phar://phpstan.phar/src/Analyser/ExprHandler/MethodCallHandler.php(136): PHPStan\Analyser\NodeScopeResolver->processArgs()
+#15 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(2216): PHPStan\Analyser\ExprHandler\MethodCallHandler->processExpr()
+#16 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(900): PHPStan\Analyser\NodeScopeResolver->processExprNode()
+#17 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(609): PHPStan\Analyser\NodeScopeResolver->processStmtNode()
+#18 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(583): PHPStan\Analyser\NodeScopeResolver->processStmtNodesInternalWithoutFlushingPendingFibers()
+#19 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(820): PHPStan\Analyser\NodeScopeResolver->processStmtNodesInternal()
+#20 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(609): PHPStan\Analyser\NodeScopeResolver->processStmtNode()
+#21 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(583): PHPStan\Analyser\NodeScopeResolver->processStmtNodesInternalWithoutFlushingPendingFibers()
+#22 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(1034): PHPStan\Analyser\NodeScopeResolver->processStmtNodesInternal()
+#23 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(609): PHPStan\Analyser\NodeScopeResolver->processStmtNode()
+#24 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(583): PHPStan\Analyser\NodeScopeResolver->processStmtNodesInternalWithoutFlushingPendingFibers()
+#25 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(972): PHPStan\Analyser\NodeScopeResolver->processStmtNodesInternal()
+#26 phar://phpstan.phar/src/Analyser/NodeScopeResolver.php(458): PHPStan\Analyser\NodeScopeResolver->processStmtNode()
+#27 phar://phpstan.phar/src/Analyser/FileAnalyser.php(122): PHPStan\Analyser\NodeScopeResolver->processNodes()
+#28 phar://phpstan.phar/src/Analyser/Analyser.php(81): PHPStan\Analyser\FileAnalyser->analyseFile()
+#29 phar://phpstan.phar/src/Command/AnalyserRunner.php(70): PHPStan\Analyser\Analyser->analyse()
+#30 phar://phpstan.phar/src/Command/AnalyseApplication.php(188): PHPStan\Command\AnalyserRunner->runAnalyser()
+#31 phar://phpstan.phar/src/Command/AnalyseApplication.php(80): PHPStan\Command\AnalyseApplication->runAnalyser()
+#32 phar://phpstan.phar/src/Command/AnalyseCommand.php(263): PHPStan\Command\AnalyseApplication->analyse()
+#33 phar://phpstan.phar/vendor/symfony/console/Command/Command.php(259): PHPStan\Command\AnalyseCommand->execute()
+#34 phar://phpstan.phar/vendor/symfony/console/Application.php(868): _PHPStan_b3f880679\Symfony\Component\Console\Command\Command->run()
+#35 phar://phpstan.phar/vendor/symfony/console/Application.php(261): _PHPStan_b3f880679\Symfony\Component\Console\Application->doRunCommand()
+#36 phar://phpstan.phar/vendor/symfony/console/Application.php(157): _PHPStan_b3f880679\Symfony\Component\Console\Application->doRun()
+#37 phar://phpstan.phar/bin/phpstan(109): _PHPStan_b3f880679\Symfony\Component\Console\Application->run()
+#38 phar://phpstan.phar/bin/phpstan(110): {closure:phar://phpstan.phar/bin/phpstan:16}()
+#39 @ROOT@/vendor/phpstan/phpstan/phpstan(8): require('...')
+#40 @ROOT@/vendor/bin/phpstan(119): include('...')
+#41 {main}
+@exit@ 1
